colorcoder/Main.java

```diff
@@ -1,120 +1,38 @@
 package colorcoder;
 
```

Comment:
same name used for Class and Method( Main)
Main class holds too many responsibilities